projects/1.19/assets/such-a-delight/suchadelight/lang/zh_cn.json

```
@@ -0,0 +1,234 @@
+  "item.suchadelight.corn_pudding": "玉米布丁",
+  "item.suchadelight.cornbread": "玉米面包",
+  "item.suchadelight.corned_beef": "咸牛肉",
+  "item.suchadelight.crimson_cheese_slice": "绯红芝士片",
+  "block.suchadelight.crimson_cheese_wheel": "绯红芝士圈",
+  "item.suchadelight.crimson_tea": "绯红茶",
+  "item.suchadelight.egg_salad": "鸡蛋沙拉",
+  "item.suchadelight.eggplant_bolognese": "茄子肉酱",
+  "item.suchadelight.eggplant_burger": "茄子汉堡",
+  "item.suchadelight.eggplant_lasagna": "茄子千层面",
+  "item.suchadelight.eggplant_parmesan": "茄子帕尔马干酪",
+  "item.suchadelight.elotes": "Elotes",
+  "item.suchadelight.enchilada": "辣酱玉米饼馅",
+  "item.suchadelight.fajitas": "法吉塔斯",
+  "item.suchadelight.falafel": "三明治",
+  "item.suchadelight.fish_taco": "鱼炸玉米饼",
+  "item.suchadelight.french_toast": "法式吐司",
+  "item.suchadelight.fried_chicken_dinner_plate": "炸鸡餐盘",
+  "item.suchadelight.fruit_tart": "水果馅饼",
+  "item.suchadelight.gilded_tea": "镀金茶",
+  "item.suchadelight.granola": "麦片",
+  "item.suchadelight.grilled_corn": "烤玉米",
+  "item.suchadelight.grilled_eggplant": "烤茄子",
+  "item.suchadelight.gyros": "陀螺",
+  "item.suchadelight.honey_tea": "蜂蜜茶",
+  "item.suchadelight.hot_dog": "热狗",
+  "item.suchadelight.hummus": "鹰嘴豆泥",
+  "item.suchadelight.kimchi": "泡菜",
+  "item.suchadelight.kippers": "腌鱼",
+  "item.suchadelight.mayonnaise": "蛋黄酱",
+  "item.suchadelight.minestrone_soup": "通心粉汤",
+  "item.suchadelight.moussaka": "穆萨卡",
+  "item.suchadelight.muffuletta": "穆富莱塔",
+  "item.suchadelight.mystic_jam_blue": "神秘蓝色果酱",
+  "item.suchadelight.mystic_wine_blue": "神秘蓝色果酒",
+  "item.suchadelight.mystic_jam_black": "神秘黑色果酱",
+  "item.suchadelight.mystic_wine_black": "神秘黑色果酒",
+  "item.suchadelight.mystic_jam_red": "神秘红色果酱",
+  "item.suchadelight.mystic_wine_red": "神秘红色果酒",
+  "item.suchadelight.mystic_jam_gold": "神秘金色果酱",
+  "item.suchadelight.mystic_wine_gold": "神秘金色果酒",
+  "item.suchadelight.mystic_jam_white": "神秘白色果酱",
+  "item.suchadelight.mystic_wine_white": "神秘白色果酒",
+  "item.suchadelight.natto": "纳豆",
+  "item.suchadelight.oil": "油",
+  "item.suchadelight.olive_tapenade": "橄榄塔佩纳德",
+  "item.suchadelight.pasta_salad": "意大利面沙拉",
+  "item.suchadelight.pbnj": "花生果酱三明治",
+  "item.suchadelight.peanut_butter_cookies": "花生酱曲奇",
+  "item.suchadelight.peanut_butter": "花生酱",
+  "item.suchadelight.peanut_milk": "花生奶",
+  "item.suchadelight.pepper_popper": "胡椒爆米花",
+  "item.suchadelight.pepper_steak": "胡椒牛排",
+  "item.suchadelight.pickle_chips": "泡菜片",
+  "item.suchadelight.pickled_beets": "腌甜菜",
+  "item.suchadelight.pickled_corn": "腌玉米",
+  "item.suchadelight.pickled_peppers": "泡椒",
+  "item.suchadelight.pickles": "腌菜",
+  "item.suchadelight.pina_colada": "凤梨可乐达",
+  "item.suchadelight.pineapple_chicken_skewer": "菠萝鸡肉串",
+  "item.suchadelight.pineapple_curry_rice_serving": "菠萝咖喱饭",
+  "item.suchadelight.pineapple_salsa": "菠萝莎莎酱",
+  "item.suchadelight.pineapple_slices": "菠萝片",
+  "item.suchadelight.pizza_slice": "披萨片",
+  "item.suchadelight.pizza_tower_take": "披萨塔服务",
```

serving指“一份”
这个在游戏中是怎样获得的
```suggestion
  "item.suchadelight.pizza_tower_take": "单份披萨塔",
```

---

> serving指“一份” 这个在游戏中是怎样获得的

三张披萨+铁栅栏+其他，确实是塔